docs/methods/cardanoSignTransaction.md

```diff
@@ -191,8 +190,48 @@ TrezorConnect.cardanoSignTransaction({
 });
 ```
 
+#### Catalyst voting key registration
+```javascript
+TrezorConnect.cardanoSignTransaction({
+    inputs: [
+        {
+            path: "m/1852'/1815'/0'/0/0",
+            prev_hash: "3b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b7",
+            prev_index: 0,
+        }
+    ],
+    outputs: [
+        {
+            address: "addr1q84sh2j72ux0l03fxndjnhctdg7hcppsaejafsa84vh7lwgmcs5wgus8qt4atk45lvt4xfxpjtwfhdmvchdf2m3u3hlsd5tq5r",
+            amount: "3003112",
+        }
+    ],
+    fee: "42",
+    ttl: "10",
+    auxiliaryData: {
+        type: 1,
```

Comment:
just wondering - doesn't connect expose the enums in some more "developer-friendly" way? I'd add at least in a comment next to the type what's it's meaning

Reply:
I added export of those enums to the type files. However, I decided not to use them in the docs, since it makes sense that pure Javascript is used there. So I added a description next to the numbers - as you mentioned.